src/utils/auth.ts

```diff
@@ -104,6 +107,32 @@ export default class Auth {
     }
   }
 
+  handleCookieBasedAccessTokenAuthentication(
+    accessToken: string,
+    expiresInSeconds: string,
+  ) {
+    // handle any previous location redirects here
```

Comment:
On second thought, this is probably not where previous URL redirects will happen.